### PR TITLE
Add TypeScript types

### PR DESCRIPTION
**Pull request by lapkoshka**
image

### CODE REVIEW

File: types.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+declare module 'near-seed-phrase' {
+    interface Seed {
+        seedPhrase: string;
+        secretKey: string;
+        publicKey: string;
+    }
+
+    export function parseSeedPhrase(phrase: string): Seed;
```
Comment (miraclx) [inline types.d.ts]:
```suggestion
    export function parseSeedPhrase(seedPhrase: string, derivationPath: string): Seed;
```

**Review by miraclx — CHANGES_REQUESTED** [review]
Let's define everything to more appropriately match the exports and their signatures.

https://github.com/near/near-seed-phrase/blob/6e885ec6e77428bf88e5aca2b391f7009c89029e/index.js#L32-L38

**Comment by miraclx** [comment]
@loosefer, you still have to include signature exports for the other functions. - `normalizeSeedPhrase` and `findSeedPhraseKey`.

**Comment by lapkoshka** [comment]
@miraclx Excuse me, i'm hurried before clicked review, now it's done :)

**Review by miraclx — APPROVED** [review]
Awesome, looks good to me. Thanks!

**Pull request merged**